### PR TITLE
Skills section

### PR DESCRIPTION
**Pull request by RalucaRbc**

### CODE REVIEW

File: src/_component/skills/skills.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+//  {
+//   font-size: 25px;  
+// }
+
+.skill__icon i[class^="icon__"] {
+  height: 30px;
+  width: 30px;
+}
```
Comment (MariaTeo) [inline src/_component/skills/skills.scss:42]: Lines 47,50,53 need an empty line

File: src/_component/skills/skills.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  color: #D52027;
+}
+i[class="icon__trophy"] {
+  color: #50D166;
+}
+i[class="icon__bell"] {
+  color: #F18F1C;
+}
+
+
```
Comment (MariaTeo) [inline src/_component/skills/skills.scss:59]: No need of 3 empty lines at the end 😬

File: src/_asset/font/font.scss [hunk 3]
```diff
@@ -26,6 +26,11 @@
   -moz-osx-font-smoothing: grayscale;
 }
 
+.icon__bell {
+  &:before {
+    content: $icon__bell; 
+  }
+}
```
Comment (MariaTeo) [inline src/_asset/font/font.scss:33]: Empty lines before and after for readability. 
Ps. Pls don't be mad at me 😅😌

**Review by MariaTeo — COMMENTED** [review]
Nice work

File: src/_asset/font/font.scss (hunk 3 above)
Comment (RalucaRbc) [reply src/_asset/font/font.scss:33]: It's WIP and it's been saved as a draft 😃 🤗

Comment (MariaTeo) [reply src/_asset/font/font.scss:33]: I know, i saw that.

**Review by NicolaePaulGeambazu — APPROVED** [review]
Nice Work

**Review by BogdanNegura — APPROVED** [review]
💯 : Nice job!

**Review by marianzburlea — APPROVED** [review]
mega super 💯

**Pull request merged**